README.md

```diff
@@ -18,7 +18,8 @@ This API responds to the below endpoints:
   * This endpoint uses Incognia API to decide between frictionlessly sign in the user or sending an OTP through email.
 * Validate signin OTP: `POST /signin/validate_otp`
 * Validate signin QR code (mobile supported Web login): `POST /signin/validate_qrcode`
-* Assess user login and signup: `POST /assessments/assess`
+* Assess user login and signup: `POST /users/:user_id/assessments/assess`
+* Return user latest assessments: `GET /users/:user_id/assessments/latest`
```

Comment:
I couldn't config this to appear `account_id`.

Reply:
Does it throw an error?

Reply:
Nop, Igaz. This is only internally, the user just build the URL and send the request.